src/utils/renderChunk.jsx

```diff
@@ -17,9 +17,9 @@ export const renderChunk = (data, props) => {
     case "long_text":
       return (<em-span
         {...defaultprops}
+        dangerouslySetInnerHTML={{__html: parsedChunk}}
         {...props}
       >
-        {parsedChunk}
       </em-span>);
```

Comment:
How bout self closing tag since it has no children anymore?

Kinda like:
```js
return (
  <em-span
      {...defaultprops}
      dangerouslySetInnerHTML={{__html: parsedChunk}}
      {...props}
  />
);
```